great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -38,9 +39,16 @@
     )
     from great_expectations.render.renderer_configuration import AddParamArgs
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the set of distinct column values to be contained by a given set."
+)
+VALUE_SET_ARG = "A set of objects used for comparison."
```

Comment:
Why are the other fields called "DESCRIPTION", but this one is called "ARG"?

Reply:
Ok, but in the next PR you are going to be adding this "arg" to the pydantic description.